src/Agent.Worker/ExecutionContext.cs

```diff
@@ -142,14 +134,6 @@ public void Start(string currentOperation = null, TimeSpan? timeout = null)
             _record.State = TimelineRecordState.InProgress;
 
             _jobServerQueue.QueueTimelineRecordUpdate(_mainTimelineId, _record);
-
-            if (timeout != null)
```

Comment:
you take the whole section out of using "timeout" but still live it as a input arg at Start();